frida_mode/src/cmplog/cmplog_x64.c

```diff
@@ -256,6 +254,9 @@ static void cmplog_instrument_cmp_sub(const cs_insn *     instr,
   if (operand1->type == X86_OP_INVALID) return;
   if (operand2->type == X86_OP_INVALID) return;
 
+  if (operand1->size != operand2->size) FATAL("Operand size mismatch");
```

Comment:
why fatal?

Reply:
It should never happen. It's more a sanity check. It's only done on block compilation though. Would you prefer I remove it?

Reply:
yes. sanity checks are only a waste of time in fuzzing :)

Reply:
Done.

Reply:
leaving sanity checks in place for debug builds would/could make sense?